requirements.txt

```diff
@@ -18,6 +18,7 @@ django-slack==5.17.8
 django-tagging==0.5.0
 django-watson==1.6.2
 django-prometheus==2.2.0
+django-rest-multiple-models==2.1.3
```

Comment:
Where is this used in this PR?

Reply:
It wasn't!...but I kept getting errors while running the app until I added that. Looks like a rebase may have fixed it. I'll remove it and repush.